domains/azizulabedinazmi.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "Documentation website for is-a.dev",
+  "repo": "https://github.com/is-a-dev/docs",
+  "owner": {
+    "username": "azizulabedinazmi",
+    "email": "admin@is-a.dev"
+  },
+  "record": {
+    "URL": "https://github.com/azizulabedinazmi"
+  },
+  "proxied": true
```

URL records are proxied by default
```suggestion
    "URL": "https://github.com/azizulabedinazmi"
```